synth.py

```diff
@@ -29,4 +29,3 @@
   '.kokoro/continuous/node10/system-test.cfg',
```

Comment:
Do we want a synthfile here?

Reply:
Let's hold off on `synth.py` for now, I'd rather use actions.